test/demo/check-demo.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")"
+cd ../..
+
+check () {
```

Comment:
The script may not actually error, it may just have different output, so this tests the output is exactly as expected